sites/footers/reports/a11y-1.json

```diff
@@ -0,0 +1,2032 @@
SIsInJlcXVpcmVtZW50RGVzY3JpcHRpb24iOiJBUklBIGF0dHJpYnV0ZXMgbXVzdCBjb25mb3JtIHRvIHZhbGlkIHZhbHVlcyJ9LHsicmVxdWlyZW1lbnRLZXkiOiJhcmlhLXZhbGlkLWF0dHIiLCJyZXF1aXJlbWVudERlc2NyaXB0aW9uIjoiQVJJQSBhdHRyaWJ1dGVzIG11c3QgY29uZm9ybSB0byB2YWxpZCBuYW1lcyJ9LHsicmVxdWlyZW1lbnRLZXkiOiJidXR0b24tbmFtZSIsInJlcXVpcmVtZW50RGVzY3JpcHRpb24iOiJCdXR0b25zIG11c3QgaGF2ZSBkaXNjZXJuaWJsZSB0ZXh0In0seyJyZXF1aXJlbWVudEtleSI6ImZyYW1lLXRpdGxlIiwicmVxdWlyZW1lbnREZXNjcmlwdGlvbiI6IkZyYW1lcyBtdXN0IGhhdmUgYW4gYWNjZXNzaWJsZSBuYW1lIn0seyJyZXF1aXJlbWVudEtleSI6ImlucHV0LWJ1dHRvbi1uYW1lIiwicmVxdWlyZW1lbnREZXNjcmlwdGlvbiI6IklucHV0IGJ1dHRvbnMgbXVzdCBoYXZlIGRpc2Nlcm5pYmxlIHRleHQifSx7InJlcXVpcmVtZW50S2V5IjoibGFiZWwiLCJyZXF1aXJlbWVudERlc2NyaXB0aW9uIjoiRm9ybSBlbGVtZW50cyBtdXN0IGhhdmUgbGFiZWxzIn0seyJyZXF1aXJlbWVudEtleSI6ImxpbmstbmFtZSIsInJlcXVpcmVtZW50RGVzY3JpcHRpb24iOiJMaW5rcyBtdXN0IGhhdmUgZGlzY2VybmlibGUgdGV4dCJ9LHsicmVxdWlyZW1lbnRLZXkiOiJuZXN0ZWQtaW50ZXJhY3RpdmUiLCJyZXF1aXJlbWVudERlc2NyaXB0aW9uIjoiSW50ZXJhY3RpdmUgY29udHJvbHMgbXVzdCBub3QgYmUgbmVzdGVkIn0seyJyZXF1aXJlbWVudEtleSI6InNlbGVjdC1uYW1lIiwicmVxdWlyZW1lbnREZXNjcmlwdGlvbiI6IlNlbGVjdCBlbGVtZW50IG11c3QgaGF2ZSBhbiBhY2Nlc3NpYmxlIG5hbWUifSx7InJlcXVpcmVtZW50S2V5IjoibGlua0Z1bmN0aW9uIiwicmVxdWlyZW1lbnREZXNjcmlwdGlvbiI6IklmIGFuIGFuY2hvciBlbGVtZW50IGZ1bmN0aW9ucyBhcyBhIGN1c3RvbSB3aWRnZXQsIGl0IG11c3QgaGF2ZSB0aGUgYXBwcm9wcmlhdGUgQVJJQSB3aWRnZXQgcm9sZSJ9LHsicmVxdWlyZW1lbnRLZXkiOiJ3aWRnZXRGdW5jdGlvbiIsInJlcXVpcmVtZW50RGVzY3JpcHRpb24iOiJJZiBhIG5hdGl2ZSB3aWRnZXQgZnVuY3Rpb25zIGFzIGEgY3VzdG9tIHdpZGdldCwgaXQgbXVzdCBoYXZlIHRoZSBhcHByb3ByaWF0ZSBBUklBIHdpZGdldCByb2xlIn0seyJyZXF1aXJlbWVudEtleSI6Im5hdGl2ZUN1ZXMiLCJyZXF1aXJlbWVudERlc2NyaXB0aW9uIjoiSWYgYSBuYXRpdmUgd2lkZ2V0IGFkb3B0cyBjZXJ0YWluIGludGVyYWN0aXZlIHN0YXRlcywgaXQgbXVzdCBwcm92aWRlIGFwcHJvcHJpYXRlIGN1ZXMifSx7InJlcXVpcmVtZW50S2V5IjoiZGVzaWduUGF0dGVybiIsInJlcXVpcmVtZW50RGVzY3JpcHRpb24iOiJBIGN1c3RvbSB3aWRnZXQgbXVzdCBoYXZlIHRoZSBhcHByb3ByaWF0ZSBBUklBIHdpZGdldCByb2xlIGZvciBpdHMgZGVzaWduIHBhdHRlcm4ifSx7InJlcXVpcmVtZW50S2V5Ijoicm9sZVN0YXRlUHJvcGVydHkiLCJyZXF1aXJlbWVudERlc2NyaXB0aW9uIjoiQSBjdXN0b20gd2lkZ2V0IG11c3Qgc3VwcG9ydCB0aGUgQVJJQSByb2xlcywgc3RhdGVzLCBhbmQgcHJvcGVydGllcyBzcGVjaWZpZWQgYnkgaXRzIGRlc2lnbiBwYXR0ZXJuIn0seyJyZXF1aXJlbWVudEtleSI6ImN1c3RvbVdpZGdldHNDdWVzIiwicmVxdWlyZW1lbnREZXNjcmlwdGlvbiI6IklmIGEgY3VzdG9tIHdpZGdldCBhZG9wdHMgY2VydGFpbiBpbnRlcmFjdGl2ZSBzdGF0ZXMsIGl0IG11c3QgY29tbXVuaWNhdGUgdGhvc2Ugc3RhdGVzIHByb2dyYW1tYXRpY2FsbHkifSx7InJlcXVpcmVtZW50S2V5IjoiZnJhbWVUaXRsZSIsInJlcXVpcmVtZW50RGVzY3JpcHRpb24iOiJBIGZyYW1lIG9yIGlmcmFtZSBtdXN0IGhhdmUgYSB0aXRsZSB0aGF0IGRlc2NyaWJlcyBpdHMgY29udGVudCJ9XSwicmVxdWlyZW1lbnRzSW5jb21wbGV0ZSI6W119LHsid2NhZ051bWJlciI6IjQuMS4zIiwid2NhZ05hbWUiOiJTdGF0dXMgTWVzc2FnZXMiLCJzdGF0dXMiOiJwYXNzIiwicmVxdWlyZW1lbnRzRmFpbGVkIjpbXSwicmVxdWlyZW1lbnRzUGFzc2VkIjpbeyJyZXF1aXJlbWVudEtleSI6InN0YXR1c01lc3NhZ2VzIiwicmVxdWlyZW1lbnREZXNjcmlwdGlvbiI6IlN0YXR1cyBtZXNzYWdlcyBtdXN0IGJlIHByb2dyYW1tYXRpY2FsbHkgZGV0ZXJtaW5hYmxlIHdpdGhvdXQgcmVjZWl2aW5nIGZvY3VzIn1dLCJyZXF1aXJlbWVudHNJbmNvbXBsZXRlIjpbXX1dfQ=="
```

Comment:
I reviewed how to add attachment on our JSON document and here how it will look like according to the latest version

```suggestion
      "earl:mode": "earl:automatic",
      "acr:asset": {
        "@type": "acr:AttachementJson",
        "dct:title": "JSON report saved from the Accessibility Insight tool",
        "acr:content": {
          "@type": "@json",
          "@value": {"url":"https://universallabs.org/wet/labs/pr2043/sites/footers/footers-en.html#wb-cont","title":"Footer - Canada.ca","date":"2022-11-25T19:56:10.339Z","comment":"There were no failures during the assessment of the footer","version":"2.35.0","results":[{"wcagNumber":"1.1.1","wcagName":"Non-text Content","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"area-alt","requirementDescription":"Active <area> elements must have alternate text"},{"requirementKey":"aria-meter-name","requirementDescription":"ARIA meter nodes must have an accessible name"},{"requirementKey":"aria-progressbar-name","requirementDescription":"ARIA progressbar nodes must have an accessible name"},{"requirementKey":"image-alt","requirementDescription":"Images must have alternate text"},{"requirementKey":"input-image-alt","requirementDescription":"Image buttons must have alternate text"},{"requirementKey":"object-alt","requirementDescription":"<object> elements must have alternate text"},{"requirementKey":"role-img-alt","requirementDescription":"[role='img'] elements must have an alternative text"},{"requirementKey":"svg-img-alt","requirementDescription":"<svg> elements with an img role must have an alternative text"},{"requirementKey":"imageFunction","requirementDescription":"Every image must be coded as either meaningful or decorative"},{"requirementKey":"textAlternative","requirementDescription":"A meaningful image must have a text alternative that serves the equivalent purpose"},{"requirementKey":"captcha","requirementDescription":"If a CAPTCHA is used, alternative methods must be provided for both users without vision and users without hearing"},{"requirementKey":"auditoryCues","requirementDescription":"Auditory cues must be accompanied by visual cues"}],"requirementsIncomplete":[]},{"wcagNumber":"1.2.1","wcagName":"Audio-only and Video-only (Prerecorded)","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"audio-caption","requirementDescription":"<audio> elements must have a captions track"},{"requirementKey":"audioOnlyEquivalent","requirementDescription":"Pre-recorded audio-only content must be accompanied by an equivalent text alternative"},{"requirementKey":"videoOnlyEquivalent","requirementDescription":"Pre-recorded video-only content must be accompanied by an equivalent text or audio alternative"}],"requirementsIncomplete":[]},{"wcagNumber":"1.2.2","wcagName":"Captions (Prerecorded)","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"video-caption","requirementDescription":"<video> elements must have captions"},{"requirementKey":"captions","requirementDescription":"Pre-recorded multimedia content must have captions"},{"requirementKey":"noObstruction","requirementDescription":"Captions must not obscure or obstruct relevant information in the video"}],"requirementsIncomplete":[]},{"wcagNumber":"1.2.3","wcagName":"Audio Description or Media Alternative (Prerecorded)","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"audioDescription","requirementDescription":"Pre-recorded video with audio must have an audio description"}],"requirementsIncomplete":[]},{"wcagNumber":"1.2.4","wcagName":"Captions (Live)","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"liveCaptions","requirementDescription":"Captions must be provided for live (streaming) video with audio"}],"requirementsIncomplete":[]},{"wcagNumber":"1.2.5","wcagName":"Audio Description (Prerecorded)","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"audioDescription","requirementDescription":"Pre-recorded video with audio must have an audio description"},{"requirementKey":"synchronization","requirementDescription":"An audio description must be synchronized with the video content"},{"requirementKey":"noConflict","requirementDescription":"An audio description must not conflict with audible information in the sound track"}],"requirementsIncomplete":[]},{"wcagNumber":"1.3.1","wcagName":"Info and Relationships","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"aria-allowed-role","requirementDescription":"ARIA role should be appropriate for the element"},{"requirementKey":"aria-hidden-focus","requirementDescription":"ARIA hidden element must not contain focusable elements"},{"requirementKey":"aria-required-children","requirementDescription":"Certain ARIA roles must contain particular children"},{"requirementKey":"aria-required-parent","requirementDescription":"Certain ARIA roles must be contained by particular parents"},{"requirementKey":"definition-list","requirementDescription":"<dl> elements must only directly contain properly-ordered <dt> and <dd> groups, <script>, <template> or <div> elements"},{"requirementKey":"dlitem","requirementDescription":"<dt> and <dd> elements must be contained by a <dl>"},{"requirementKey":"label","requirementDescription":"Form elements must have labels"},{"requirementKey":"list","requirementDescription":"<ul> and <ol> must only directly contain <li>, <script> or <template> elements"},{"requirementKey":"listitem","requirementDescription":"<li> elements must be contained in a <ul> or <ol>"},{"requirementKey":"presentation-role-conflict","requirementDescription":"Elements of role none or presentation should be flagged"},{"requirementKey":"select-name","requirementDescription":"Select element must have an accessible name"},{"requirementKey":"td-headers-attr","requirementDescription":"Table cells that use the headers attribute must only refer to cells in the same table"},{"requirementKey":"th-has-data-cells","requirementDescription":"Table headers in a data table must refer to data cells"},{"requirementKey":"landmark-roles","requirementDescription":"A landmark region must have the role that best describes its content"},{"requirementKey":"primary-content","requirementDescription":"The main landmark must contain all of the page's primary content"},{"requirementKey":"no-repeating-content","requirementDescription":"The main landmark must not contain any blocks of content that repeat across pages"},{"requirementKey":"headingFunction","requirementDescription":"An element coded as a heading must function as a heading"},{"requirementKey":"missingHeadings","requirementDescription":"Text that looks like a heading must be coded as a heading"},{"requirementKey":"headingLevel","requirementDescription":"A heading's programmatic level must match the level that's presented visually"},{"requirementKey":"nativeInstructions","requirementDescription":"If a native widget has a visible label or instructions, they must be programmatically determinable"},{"requirementKey":"nativeCues","requirementDescription":"If a native widget adopts certain interactive states, it must provide appropriate cues"},{"requirementKey":"customWidgetsInstructions","requirementDescription":"If a custom widget has a visible label or instructions, they must be programmatically determinable"},{"requirementKey":"customWidgetsCues","requirementDescription":"If a custom widget adopts certain interactive states, it must communicate those states programmatically"},{"requirementKey":"columns","requirementDescription":"Content presented in multi-column format must support a correct reading sequence"},{"requirementKey":"cssContent","requirementDescription":"Meaningful content must not be implemented using only CSS :before or :after"},{"requirementKey":"tableSemantics","requirementDescription":"A undefined element must be coded correctly as a data table or a layout table"},{"requirementKey":"headers","requirementDescription":"Coded headers must be used correctly"},{"requirementKey":"headersAttribute","requirementDescription":"The headers attribute of a undefined element must reference the correct undefined element(s)"},{"requirementKey":"lists","requirementDescription":"Lists must be contained within semantically correct containers"},{"requirementKey":"emphasis","requirementDescription":"Words and phrases that are visually emphasized must be contained within semantically correct containers"},{"requirementKey":"quotes","requirementDescription":"The undefined element must not be used to style non-quote text"},{"requirementKey":"letterSpacing","requirementDescription":"Spacing characters must not be used to increase the space between letters in a word"}],"requirementsIncomplete":[]},{"wcagNumber":"1.3.2","wcagName":"Meaningful Sequence","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"textDirection","requirementDescription":"If a page or a passage uses a script that is read right-to-left, it must have the correct text direction"},{"requirementKey":"cssPositioning","requirementDescription":"Meaningful content positioned on the page using CSS must retain its meaning when CSS is disabled"},{"requirementKey":"layoutTables","requirementDescription":"The content in an HTML layout table must make sense when the table is linearized"},{"requirementKey":"columns","requirementDescription":"Content presented in multi-column format must support a correct reading sequence"}],"requirementsIncomplete":[]},{"wcagNumber":"1.3.3","wcagName":"Sensory Characteristics","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"sensoryCharacteristics","requirementDescription":"Instructions must not rely solely on color or other sensory characteristics"}],"requirementsIncomplete":[]},{"wcagNumber":"1.3.4","wcagName":"Orientation","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"orientation","requirementDescription":"Web content must not be locked to a particular screen orientation"}],"requirementsIncomplete":[]},{"wcagNumber":"1.3.5","wcagName":"Identify Input Purpose","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"autocomplete-valid","requirementDescription":"autocomplete attribute must be used correctly"},{"requirementKey":"nativeAutocomplete","requirementDescription":"Text fields that serve certain purposes must have the correct HTML5 autocomplete attribute"}],"requirementsIncomplete":[]},{"wcagNumber":"1.4.1","wcagName":"Use of Color","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"useOfColor","requirementDescription":"Color must not be used as the only visual means for conveying meaning"}],"requirementsIncomplete":[]},{"wcagNumber":"1.4.10","wcagName":"Reflow","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"reflow","requirementDescription":"Content must be visible without having to scroll in two dimensions"}],"requirementsIncomplete":[]},{"wcagNumber":"1.4.11","wcagName":"Non-text Contrast","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"ui-components","requirementDescription":"Visual information used to identify active user interface components and their states must have sufficient contrast"},{"requirementKey":"state-changes","requirementDescription":"Any visual information that indicates a component's state must have sufficient contrast"},{"requirementKey":"graphics","requirementDescription":"Graphics must have sufficient contrast"}],"requirementsIncomplete":[]},{"wcagNumber":"1.4.12","wcagName":"Text Spacing","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"avoid-inline-spacing","requirementDescription":"Inline text spacing must be adjustable with custom stylesheets"},{"requirementKey":"textSpacing","requirementDescription":"Users must be able to adjust text spacing with no loss of content or functionality"}],"requirementsIncomplete":[]},{"wcagNumber":"1.4.13","wcagName":"Content on Hover or Focus","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"hoverFocusContent","requirementDescription":"Content that appears on focus or hover must be dismissible, hoverable, and persistent"}],"requirementsIncomplete":[]},{"wcagNumber":"1.4.2","wcagName":"Audio Control","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"audio-control","requirementDescription":"If audio content plays automatically for longer than three seconds, users must be able to pause or mute it"}],"requirementsIncomplete":[]},{"wcagNumber":"1.4.3","wcagName":"Contrast (Minimum)","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"color-contrast","requirementDescription":"Elements must have sufficient color contrast"},{"requirementKey":"contrast","requirementDescription":"Text elements must have sufficient contrast"}],"requirementsIncomplete":[]},{"wcagNumber":"1.4.4","wcagName":"Resize text","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"resizeText","requirementDescription":"Users must be able to resize text, without using assistive technology, up to 200% with no loss of content or functionality"}],"requirementsIncomplete":[]},{"wcagNumber":"1.4.5","wcagName":"Images of Text","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"imageOfText","requirementDescription":"Images of text are allowed only where a specific appearance is required (e.g., logotypes)"}],"requirementsIncomplete":[]},{"wcagNumber":"2.1.1","wcagName":"Keyboard","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"frame-focusable-content","requirementDescription":"Frames with focusable content must not have tabindex=-1"},{"requirementKey":"server-side-image-map","requirementDescription":"Server-side image maps must not be used"},{"requirementKey":"keyboard-navigation","requirementDescription":"Users must be able to navigate to all interactive interface components using a keyboard"},{"requirementKey":"on-keystroke-timing","requirementDescription":"Components must not require specific timings for individual keystrokes"},{"requirementKey":"keyboardInteraction","requirementDescription":"A custom widget must support the keyboard interaction specified by its design pattern"}],"requirementsIncomplete":[]},{"wcagNumber":"2.1.2","wcagName":"No Keyboard Trap","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"no-keyboard-traps","requirementDescription":"Users must be able to navigate away from all components using a keyboard"}],"requirementsIncomplete":[]},{"wcagNumber":"2.1.4","wcagName":"Character Key Shortcuts","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"character-key-shortcuts","requirementDescription":"Users must be able to turn off or remap character key shortcuts"}],"requirementsIncomplete":[]},{"wcagNumber":"2.2.1","wcagName":"Timing Adjustable","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"meta-refresh","requirementDescription":"Timed refresh must not exist"},{"requirementKey":"time-limits","requirementDescription":"If a time limit is set by the content, the user must be able to turn off, adjust, or extend the time limit"}],"requirementsIncomplete":[]},{"wcagNumber":"2.2.2","wcagName":"Pause, Stop, Hide","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"blink","requirementDescription":"<blink> elements are deprecated and must not be used"},{"requirementKey":"marquee","requirementDescription":"<marquee> elements are deprecated and must not be used"},{"requirementKey":"moving-content","requirementDescription":"If content moves, blinks, or scrolls automatically for more than five seconds, users must be able to pause, stop, or hide it"},{"requirementKey":"auto-updating-content","requirementDescription":"If content updates automatically, users must be able to pause, stop, hide, or control frequency of the updates"}],"requirementsIncomplete":[]},{"wcagNumber":"2.3.1","wcagName":"Three Flashes or Below Threshold","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"flashing","requirementDescription":"Web pages must not have content that flashes more than three times per second"}],"requirementsIncomplete":[]},{"wcagNumber":"2.4.1","wcagName":"Bypass Blocks","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"bypass","requirementDescription":"Page must have means to bypass repeated blocks"},{"requirementKey":"frame-title","requirementDescription":"Frames must have an accessible name"},{"requirementKey":"primary-content","requirementDescription":"The main landmark must contain all of the page's primary content"},{"requirementKey":"no-repeating-content","requirementDescription":"The main landmark must not contain any blocks of content that repeat across pages"},{"requirementKey":"bypassBlocks","requirementDescription":"A page must provide a keyboard-accessible method to bypass repetitive content"}],"requirementsIncomplete":[]},{"wcagNumber":"2.4.2","wcagName":"Page Titled","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"document-title","requirementDescription":"Documents must have <title> element to aid in navigation"},{"requirementKey":"pageTitle","requirementDescription":"A web page must have a title that describes its topic or purpose"}],"requirementsIncomplete":[]},{"wcagNumber":"2.4.3","wcagName":"Focus Order","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"revealing-content","requirementDescription":"Activating a component that reveals hidden content must move input focus into the revealed content"},{"requirementKey":"modal-dialogs","requirementDescription":"Users must not be able to Tab away from a modal dialog without explicitly dismissing it"},{"requirementKey":"closing-content","requirementDescription":"Closing revealed content must return input focus to the component that revealed it"},{"requirementKey":"focus-order","requirementDescription":"Components must receive focus in an order that preserves meaning and operability"}],"requirementsIncomplete":[]},{"wcagNumber":"2.4.4","wcagName":"Link Purpose (In Context)","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"area-alt","requirementDescription":"Active <area> elements must have alternate text"},{"requirementKey":"link-name","requirementDescription":"Links must have discernible text"},{"requirementKey":"linkPurpose","requirementDescription":"The purpose of a link must be described by its link text alone, or by the link text together with preceding page context"}],"requirementsIncomplete":[]},{"wcagNumber":"2.4.5","wcagName":"Multiple Ways","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"generalNavigation","requirementDescription":"Users must have multiple ways to navigate to a page"}],"requirementsIncomplete":[]},{"wcagNumber":"2.4.6","wcagName":"Headings and Labels","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"headingFunction","requirementDescription":"An element coded as a heading must function as a heading"},{"requirementKey":"missingHeadings","requirementDescription":"Text that looks like a heading must be coded as a heading"}],"requirementsIncomplete":[]},{"wcagNumber":"2.4.7","wcagName":"Focus Visible","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"visible-focus","requirementDescription":"Components must provide a visible indication when they have the input focus"}],"requirementsIncomplete":[]},{"wcagNumber":"2.5.1","wcagName":"Pointer Gestures","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"pointer-gestures","requirementDescription":"Functions must be operable without requiring multipoint or path-based gestures"}],"requirementsIncomplete":[]},{"wcagNumber":"2.5.2","wcagName":"Pointer Cancellation","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"pointer-cancellation","requirementDescription":"Users must be able to cancel functions that can be operated using a single pointer"}],"requirementsIncomplete":[]},{"wcagNumber":"2.5.3","wcagName":"Label in Name","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"labelInName","requirementDescription":"A link's accessible name must contain its visible text label"},{"requirementKey":"nativeInstructions","requirementDescription":"If a native widget has a visible label or instructions, they must be programmatically determinable"},{"requirementKey":"customWidgetsInstructions","requirementDescription":"If a custom widget has a visible label or instructions, they must be programmatically determinable"}],"requirementsIncomplete":[]},{"wcagNumber":"2.5.4","wcagName":"Motion Actuation","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"motion-operation","requirementDescription":"If a function can be operated through motion, it must also be operable through user interface components"}],"requirementsIncomplete":[]},{"wcagNumber":"3.1.1","wcagName":"Language of Page","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"html-has-lang","requirementDescription":"<html> element must have a lang attribute"},{"requirementKey":"html-lang-valid","requirementDescription":"<html> element must have a valid value for the lang attribute"},{"requirementKey":"html-xml-lang-mismatch","requirementDescription":"HTML elements with lang and xml:lang must have the same base language"},{"requirementKey":"languageOfPage","requirementDescription":"A page must have the correct default language"}],"requirementsIncomplete":[]},{"wcagNumber":"3.1.2","wcagName":"Language of Parts","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"valid-lang","requirementDescription":"lang attribute must have a valid value"},{"requirementKey":"languageOfParts","requirementDescription":"If the language of a passage differs from the default language of the page, the passage must have its own language attribute"}],"requirementsIncomplete":[]},{"wcagNumber":"3.2.1","wcagName":"On Focus","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"on-focus","requirementDescription":"Navigating to a component must not trigger any unexpected change of context"}],"requirementsIncomplete":[]},{"wcagNumber":"3.2.2","wcagName":"On Input","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"on-input","requirementDescription":"Changing a component's settings must not trigger any unexpected change of context"}],"requirementsIncomplete":[]},{"wcagNumber":"3.2.3","wcagName":"Consistent Navigation","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"consistentNavigation","requirementDescription":"Navigational mechanisms that appear on multiple pages must be presented in the same relative order"}],"requirementsIncomplete":[]},{"wcagNumber":"3.2.4","wcagName":"Consistent Identification","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"consistentIdentification","requirementDescription":"Functional components that appear on multiple pages must be identified consistently"}],"requirementsIncomplete":[]},{"wcagNumber":"3.3.1","wcagName":"Error Identification","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"errorIdentification","requirementDescription":"If an input error is automatically detected, the item in error must be identified, and the error described, in text"}],"requirementsIncomplete":[]},{"wcagNumber":"3.3.2","wcagName":"Labels or Instructions","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"nativeLabel","requirementDescription":"A native widget must have a label and/or instructions that identify the expected input"},{"requirementKey":"customWidgetsLabel","requirementDescription":"A custom widget must have a label and/or instructions that identify the expected input"}],"requirementsIncomplete":[]},{"wcagNumber":"3.3.3","wcagName":"Error Suggestion","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"errorSuggestion","requirementDescription":"If an input error is automatically detected, guidance for correcting the error must be provided"}],"requirementsIncomplete":[]},{"wcagNumber":"3.3.4","wcagName":"Error Prevention (Legal, Financial, Data)","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"errorPrevention","requirementDescription":"If submitting data might have serious consequences, users must be able to correct the data input before finalizing a submission"}],"requirementsIncomplete":[]},{"wcagNumber":"4.1.1","wcagName":"Parsing","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"duplicate-id-active","requirementDescription":"IDs of active elements must be unique"},{"requirementKey":"duplicate-id-aria","requirementDescription":"IDs used in ARIA and labels must be unique"},{"requirementKey":"parsing","requirementDescription":"Elements must have complete start and end tags, must not contain duplicate attributes, and must be nested according to their specifications"}],"requirementsIncomplete":[]},{"wcagNumber":"4.1.2","wcagName":"Name, Role, Value","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"area-alt","requirementDescription":"Active <area> elements must have alternate text"},{"requirementKey":"aria-allowed-attr","requirementDescription":"Elements must only use allowed ARIA attributes"},{"requirementKey":"aria-allowed-role","requirementDescription":"ARIA role should be appropriate for the element"},{"requirementKey":"aria-command-name","requirementDescription":"ARIA commands must have an accessible name"},{"requirementKey":"aria-hidden-body","requirementDescription":"aria-hidden='true' must not be present on the document body"},{"requirementKey":"aria-hidden-focus","requirementDescription":"ARIA hidden element must not contain focusable elements"},{"requirementKey":"aria-input-field-name","requirementDescription":"ARIA input fields must have an accessible name"},{"requirementKey":"aria-required-attr","requirementDescription":"Required ARIA attributes must be provided"},{"requirementKey":"aria-roledescription","requirementDescription":"aria-roledescription must be on elements with a semantic role"},{"requirementKey":"aria-roles","requirementDescription":"ARIA roles used must conform to valid values"},{"requirementKey":"aria-toggle-field-name","requirementDescription":"ARIA toggle fields must have an accessible name"},{"requirementKey":"aria-tooltip-name","requirementDescription":"ARIA tooltip nodes must have an accessible name"},{"requirementKey":"aria-valid-attr-value","requirementDescription":"ARIA attributes must conform to valid values"},{"requirementKey":"aria-valid-attr","requirementDescription":"ARIA attributes must conform to valid names"},{"requirementKey":"button-name","requirementDescription":"Buttons must have discernible text"},{"requirementKey":"frame-title","requirementDescription":"Frames must have an accessible name"},{"requirementKey":"input-button-name","requirementDescription":"Input buttons must have discernible text"},{"requirementKey":"label","requirementDescription":"Form elements must have labels"},{"requirementKey":"link-name","requirementDescription":"Links must have discernible text"},{"requirementKey":"nested-interactive","requirementDescription":"Interactive controls must not be nested"},{"requirementKey":"select-name","requirementDescription":"Select element must have an accessible name"},{"requirementKey":"linkFunction","requirementDescription":"If an anchor element functions as a custom widget, it must have the appropriate ARIA widget role"},{"requirementKey":"widgetFunction","requirementDescription":"If a native widget functions as a custom widget, it must have the appropriate ARIA widget role"},{"requirementKey":"nativeCues","requirementDescription":"If a native widget adopts certain interactive states, it must provide appropriate cues"},{"requirementKey":"designPattern","requirementDescription":"A custom widget must have the appropriate ARIA widget role for its design pattern"},{"requirementKey":"roleStateProperty","requirementDescription":"A custom widget must support the ARIA roles, states, and properties specified by its design pattern"},{"requirementKey":"customWidgetsCues","requirementDescription":"If a custom widget adopts certain interactive states, it must communicate those states programmatically"},{"requirementKey":"frameTitle","requirementDescription":"A frame or iframe must have a title that describes its content"}],"requirementsIncomplete":[]},{"wcagNumber":"4.1.3","wcagName":"Status Messages","status":"pass","requirementsFailed":[],"requirementsPassed":[{"requirementKey":"statusMessages","requirementDescription":"Status messages must be programmatically determinable without receiving focus"}],"requirementsIncomplete":[]}]}
          }
```